strapi/strapi
# Add params serializer

### What does it do?

- Adds the paramsSerializer to axios fetch client to serialize params object to a string using qs.stringify

### Why is it needed?

- To avoid string concatenation when building query params

### How to test it?

I've update everything below but not sure I got everything. Please let me know if you know some places I missed.

Check get, put, and post requests for all updated files. They should all still make the same requests with query params that are currently on main.

### TODO

- [x] Update packages/core/upload/admin/src/hooks/useAssets.js
- [x] Update packages/core/upload/admin/src/hooks/useFolder.js
- [x] Update packages/core/upload/admin/src/hooks/useFolders.js
- [x] Update packages/core/admin/admin/src/content-manager/pages/ListView/components/TableRows/index.js
- [x] Update packages/core/admin/admin/src/content-manager/components/CollectionTypeFormWrapper/index.js
- [x] Update packages/core/admin/admin/src/content-manager/components/SingleTypeFormWrapper/index.js
- [x] Update packages/core/admin/admin/src/hooks/useAdminUsers/useAdminUsers.js
- [x] Update packages/core/admin/admin/src/pages/AuthPage/components/Register/index.js
- [x] Update packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/hooks/useAuditLogsData.js
- [x] Update packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js
